Panic on `kubectl pxc logs node`

Using pxc-v0.22.0 -- looks like it crashes on `kubectl pxc logs node` command:

```
PDC2-DELL09:/tmp $ kubectl pxc logs node
panic: unable to redefine 'n' shorthand in "node" flagset: it's already used for "px-namespace" flag

goroutine 1 [running]:
github.com/portworx/pxc/vendor/github.com/spf13/pflag.(*FlagSet).AddFlag(0xc000051100, 0xc0001cec80)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/pflag/flag.go:862 +0x6d7
github.com/portworx/pxc/vendor/github.com/spf13/pflag.(*FlagSet).AddFlagSet.func1(0xc0001cec80)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/pflag/flag.go:875 +0xa8
github.com/portworx/pxc/vendor/github.com/spf13/pflag.(*FlagSet).VisitAll(0xc0001fe100, 0xc0005dfc40)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/pflag/flag.go:278 +0x94
github.com/portworx/pxc/vendor/github.com/spf13/pflag.(*FlagSet).AddFlagSet(0xc000051100, 0xc0001fe100)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/pflag/flag.go:873 +0x58
github.com/portworx/pxc/vendor/github.com/spf13/cobra.(*Command).mergePersistentFlags(0xc0001f5900)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/cobra/command.go:1572 +0x90
github.com/portworx/pxc/vendor/github.com/spf13/cobra.(*Command).InitDefaultHelpFlag(0xc0001f5900)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/cobra/command.go:968 +0x2f
github.com/portworx/pxc/vendor/github.com/spf13/cobra.(*Command).execute(0xc0001f5900, 0x22696d0, 0x0, 0x0, 0xc0001f5900, 0x22696d0)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/cobra/command.go:750 +0x57
github.com/portworx/pxc/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc0001e9680, 0x1, 0xc0005dfef0, 0x43d58a)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
github.com/portworx/pxc/vendor/github.com/spf13/cobra.(*Command).Execute(...)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/vendor/github.com/spf13/cobra/command.go:864
github.com/portworx/pxc/cmd.Main(0x0, 0xf)
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/cmd/main.go:92 +0x32
github.com/portworx/pxc/cmd.Execute()
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/cmd/main.go:79 +0x26
main.main()
	/home/lpabon/git/golang/osd/src/github.com/portworx/pxc/main.go:24 +0x20
```

--- a/pkg/cliops/logops.go
+++ b/pkg/cliops/logops.go
@@ -35,14 +35,14 @@ func AddCommonLogOptions(lc *cobra.Command) {
 	lc.Flags().BoolP("follow", "f", false, "Specify if the logs should be streamed.")
 	lc.Flags().Bool("timestamps", false, "Include timestamps on each line in the log output")
 	lc.Flags().Bool("show-pod-info", false, "Include pod info on each line in the log output")
-	lc.Flags().BoolP("previous", "p", false, "If true, print the logs for the previous instance of the container in a pod if it exists.")
+	lc.Flags().Bool("previous", false, "If true, print the logs for the previous instance of the container in a pod if it exists.")
 	lc.Flags().Bool("ignore-errors", false, "If watching / following Portworx logs, allow for any errors that occur to be non-fatal")
 	lc.Flags().Int("max-log-requests", 5, "Specify maximum number of concurrent logs to follow. Defaults to 5.")
 	lc.Flags().Int64("limit-bytes", 0, "Maximum bytes of logs to return. Defaults to no limit.")
 	lc.Flags().Int64("tail", kubernetes.NO_TAIL_LINES, "Lines of recent log file to work on. Defaults to -1, showing all log lines. All filters will be applied on top of these lines")
 	lc.Flags().String("since-time", "", "Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of since-time / since may be used.")
 	lc.Flags().Duration("since", 0, "Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of since-time / since may be used.")
-	lc.Flags().StringP("px-namespace", "n", "kube-system", "Kubernetes namespace in which Portworx is installed")
+	lc.Flags().String("px-namespace", "kube-system", "Kubernetes namespace in which Portworx is installed")
 	lc.Flags().String("filter", "", "Comma seperated list of strings to search for. Log line will be printed if any one of the strings match. Note that if --tail is specified the filter is applied on only those many lines.")
 }
 

